src/Stryker.CLI/Stryker.CLI/FileBasedInput.cs

```diff
@@ -86,6 +86,9 @@ public class FileBasedInput : IExtraData
         [JsonPropertyName("ignore-methods")]
         public string[] IgnoreMethods { get; init; }
 
+        [JsonPropertyName("output-html-report-name")]
+        public string OutputHTMLReportName { get; init; }
```

Comment:
Consider `Html`. HTML is too caps :) And that's how they usually [have](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions) it C# :)

Reply:
That's a good suggestion, I have updated all the HTML's to Html, thank you!